<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableRelationWidget</class>
 <widget class="QWidget" name="TableRelationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QgsCollapsibleGroupBox" name="grp_table">
     <property name="toolTipDuration">
      <number>-5</number>
     </property>
     <property name="title">
      <string>&lt;table_name&gt;</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lbl_public_name">
        <property name="text">
         <string>Nom public de la table (optionnel)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lne_public_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_title">
        <property name="text">
         <string>Titre de la table</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lne_title"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="lbl_abstract">
        <property name="text">
         <string>Résumé du contenu de la table</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QTextEdit" name="txt_abstract"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_keywords">
        <property name="text">
         <string>Mot clés (optionnel)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lne_keywords"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
